pages/linux/winetricks.md

```diff
@@ -0,0 +1,16 @@
+# winetricks
+
+> Manage Wine virtual Windows environments.
+> More information: <https://wiki.winehq.org/Winetricks>.
+
+- Start a graphical setup at the default Wine location:
+
+`winetricks`
+
+- Specify a custom Wine directory to run winetricks in:
+
+`WINEPREFIX={{path/to/wine_directory}} winetricks`
+
+- Install a Windows DLL or a component to the default directory:
```

Comment:
By 'a component' do mean 'Windows component', like 'Windows DLL'?  If so, we can remove 'a':
```suggestion
- Install a Windows DLL or component to the default directory:
```